netflix-roulette/src/pages/home/components/Header/components/SearchBar/SearchBar.tsx

```diff
@@ -35,6 +35,7 @@ export const SearchBar = () => {
           placeholder="What do you want to watch?"
           onChange={handleInputChange}
           value={inputValue}
+          aria-label="searchInput"
```

Comment:
If concerned about accessibility, `role="search"` [might be useful](https://www.w3.org/TR/wai-aria-practices-1.1/examples/landmarks/search.html) as well.